modules/tiles/src/tileset/tile-3d.js

```diff
@@ -435,7 +435,7 @@ export default class TileHeader {
 
   // TODO - remove anything not related to basic visibility detection
   _initializeRenderingState(header) {
-    this.depth = header.level;
+    this.depth = this.parent && this.parent.hasOwnProperty('depth') ? this.parent.depth + 1 : 0;
```

Comment:
`depth` is not used anywhere else? Is header.level not correct?
Should we add tests somewhere for `depth`?

Reply:
When i debugged header.level it was wrong. It was always 1. But now it's correct. I'll revert this changes

Reply:
header.level working only in 1.6
It is undefined in 1.7

Reply:
OK - but the `tiles` module needs to work with both 3D tiles and I3S.

The idea is that the `I3SLoader` and `Tile3DLoader` should normalize their data into a common form understood by the tiles module.

As a side note, we should create a typescript definition for the common format expected by the `tiles` module.

Reply:
this.depth = header.level || this.parent?.depth + 1 || 0;
I hope it cover all cases

Reply:
Sure. My suggestion was that ideally this should be done inside the respective loaders, but maybe that is not possible. Fine for now.